release.json

```diff
@@ -22,6 +22,24 @@
         "WINDOWS_DDNPM_SHASUM": "4ca09e6a029dc1db2f2313b6a1ebf770ee3e733dcf500c770e9cd154490d0435",
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
+    "6.24.0-rc.5": {
+        "INTEGRATIONS_CORE_VERSION": "7.24.0-rc.5",
+        "OMNIBUS_SOFTWARE_VERSION": "7.24.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.24.0-rc.1",
+        "JMXFETCH_VERSION": "0.40.1",
+        "JMXFETCH_HASH": "472ef9f1e67c79497dcbb26a25332c43212e1ca364aec36d1efddcaf24779f25",
+        "MACOS_BUILD_VERSION": "6.24.0-rc.1",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.5"
+    },
+    "7.24.0-rc.5": {
+        "INTEGRATIONS_CORE_VERSION": "7.24.0-rc.5",
+        "OMNIBUS_SOFTWARE_VERSION": "7.24.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.24.0-rc.1",
+        "JMXFETCH_VERSION": "0.40.1",
+        "JMXFETCH_HASH": "472ef9f1e67c79497dcbb26a25332c43212e1ca364aec36d1efddcaf24779f25",
+        "MACOS_BUILD_VERSION": "7.24.0-rc.1",
```

Comment:
What `MACOS_BUILD_VERSION` is used for?

Reply:
To fetch this repo: https://github.com/DataDog/datadog-agent-macos-build which is containing the build tools/scripts for the macOS version.
See https://github.com/DataDog/datadog-agent/blob/master/tasks/release.py#L609-L611